src/plugins/experimental-discord/createGraph.js

```diff
@@ -226,15 +223,17 @@ export function createGraph(
           continue;
         }
 
-        hasWeightedEmoji = true;
         const node = reactionNode(reaction, message.timestampMs, guild);
-        wg.weights.nodeWeights.set(node.address, reactionWeight);
+        if (reactionWeight != null) {
```

Comment:
nit: strict equality check to be more explicit on the intent? (!==)

Reply:
Across the codebase we use `==` when checking null, because we don't care about the difference between null and undefined. In this case, using `!==` checking would actually induce a bug, since reactionWeight is undefined when it is unset.

Our eslint config blocks `==` checking everywhere except for on null/undefined.